src/components/Forms/AddItemForm.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import { useSelector } from 'react-redux';
+import store from '../../redux/configureStore';
+
+const AddItemForm = () => {
+  const POST_CARS_DATA = 'POST_CARS_DATA';
+  const user = useSelector((el) => el.userReducer);
+  const submitHandler = (event) => {
+    event.preventDefault();
+    axios({
+      method: 'post',
+      url: 'http://127.0.0.1:3000/api/v1/cars',
+      data: {
+        brand: event.target[0].value,
+        model: event.target[1].value,
+        registration: event.target[2].value,
+        photo: event.target[3].value,
+        user_id: 1,
+      },
+      headers: {
+        'Content-Type': 'application/json',
+
```

Comment:
- [ ] @MeqdamAlqudah You have done excellently well. However, you need to fix the linters on this project....they are failing